BAZProject/MovieDetail/MovieDetailInteractor.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  MovieDetailInteractor.swift
+//  BAZProject
+//
+//  Created 1030364 on 17/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class MovieDetailInteractor: MovieDetailInteractorProtocol {
```

Comment:
Agrega `final` a esta clase
```suggestion
final class MovieDetailInteractor: MovieDetailInteractorProtocol {
```